deploy/crds/operator_v1_installation_crd.yaml

```diff
@@ -88,6 +88,14 @@ spec:
                     for pods on the Calico network. Default: 1410'
                   format: int32
                   type: integer
+                multiInterfaceMode:
+                  description: 'MultiInterfaceMode configures what will configure
+                    multiple interface per pod. Only valid for EE installations. Default:
```

Comment:
I don't think EE is appropriate to use here, should probably be `Calico Enterprise`